src/vs/workbench/api/common/extHost.protocol.ts

```diff
@@ -288,7 +289,7 @@ export interface MainThreadTextEditorsShape extends IDisposable {
 }
 
 export interface MainThreadTreeViewsShape extends IDisposable {
-	$registerTreeViewDataProvider(treeViewId: string, options: { showCollapseAll: boolean, canSelectMany: boolean, canDragAndDrop: boolean; }): Promise<void>;
+	$registerTreeViewDataProvider(treeViewId: string, options: { showCollapseAll: boolean, canSelectMany: boolean, dragAndDropController?: DragAndDropController<any>; }): Promise<void>;
```

Comment:
Seems like the `dragAndDropController` isn't used in the implementation of this method and that this can be a boolean.